site/styles/_footer.scss

```diff
@@ -41,7 +46,7 @@ footer {
 
     @include media-breakpoint-up(xl) {
       margin-right: 20px;
-      padding: 0;
+      padding: 4px 0 0;
```

Comment:
This is giving me hacky vibes... shouldn't there be a better to vertically align these elements how you want them?

Reply:
fixed